recipes/pkgconf/all/test_package/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools, RunEnvironment
+from conans.errors import ConanException
+import os
+import shutil
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+
+    def build_requirements(self):
+        self.build_requires("automake/1.16.2")
```

Comment:
How do you see requiring `m4`?
```suggestion
        self.build_requires("automake/1.16.2")
        self.build_requires("m4/1.4.18")
```
Without it I get this (I don't have m4 in my system):
```
pkgconf/1.7.3 (test package): Calling build()
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: /home/user/.conan/data/automake/1.16.2/_/_/package/258f6f05ca54813b3d7180161753df7f9ccf7e1d/bin/aclocal --force 
autom4te: need GNU m4 1.4 or later: /usr/bin/env m4
aclocal: error: echo failed with exit status: 1
```

Reply:
https://github.com/conan-io/conan-center-index/pull/2103 is adding a m4 requirement to autoconf.